packages/flutter_tools/test/create_test.dart

```diff
@@ -48,5 +48,35 @@ void main() {
     },
     // This test can take a while due to network requests.
     timeout: new Timeout(new Duration(minutes: 2)));
+
+    // Verify that we fail with an error code when the file exists.
+    testUsingContext('fails when file exists', () async {
+      ArtifactStore.flutterRoot = '../..';
+      CreateCommand command = new CreateCommand();
+      CommandRunner runner = new CommandRunner('test_flutter', '')
+        ..addCommand(command);
+      var existingFile = new File("${temp.path.toString()}/bad");
+      if (!existingFile.existsSync()) existingFile.createSync();
+      await runner.run(['create', existingFile.path])
+          .then((int code) => expect(code, equals(1)));
+    },
+    // This test can take a while due to network requests.
+    timeout: new Timeout(new Duration(minutes: 2)));
```

Comment:
here and below, you can remove the timeout param - this is only necessary for the case where this (slow) test might succeed